### DISCUSSION TITLE
when i load a world it crashes after about 10 seconds and i can move around

### DISCUSSION THREAD

**Discussion by howtfisallnamestaken** [FabricMC/fabric-api#3989] [Player support]
[### Minecraft Logs

```txt
https://mclo.gs/RCMgcFY
https://paste.gg/p/anonymous/ec9f6dd62c124c3fad61262eaf8f9bc2
```

### Description
eh links broken use what you want
_No response_](https://mclo.gs/RCMgcFY)

**Comment by Poopooracoocoo** [comment]
Open your instance's folder, go to the crash-reports folder, then share the newest crash report.

**Comment by rotgruengelb** [comment]
This appears to be because of an incompatibility between Litematica+Sodium and Valkyrien Skies.
- https://github.com/ValkyrienSkies/Valkyrien-Skies-2/issues/818
